playbooks/loki.yaml

```
@@ -6,9 +6,11 @@
   hosts: loki_servers
   become: true
   tasks:
-    - name: Install loki
+    - name: Install loki and logcli
```

Completely optional:

```suggestion
    - name: Install required packages
```